lib/_tls_wrap.js

```diff
@@ -847,9 +847,25 @@ Server.prototype.setOptions = function(options) {
   }
 
   if (options.pfx) this.pfx = options.pfx;
-  if (options.key) this.key = options.key;
+  var defaultCiphers = options.ciphers == tls.DEFAULT_CIPHERS;
+  if (!options.key) {
+    if (options.ciphers == undefined || defaultCiphers) {
```

Comment:
Yeah as I cannot put in same `if condition` 1 liner since it limits certain number of characters in 1 line.